generators/embeddable-graphic/templates/gulp/tasks/aws.js

```diff
@@ -16,6 +17,25 @@ const cfSettings = {
   wait: false,
 };
 
+const routes = {
+  routes: {
+    // Cache video and audio for 1 week on user's computer, one month in CloudFront
+    '^.*\\.(aif|iff|m3u|m4a|mid|mp3|mpa|ra|wav|wma|3g2|3gp|asf|asx|avi|flv|mov|mp4|mpg|rm|swf|vob|wmv)': {
+      cacheTime: 86400 * 7,
+      sharedCacheTime: 86400 * 30,
```

Comment:
Can we write out all instances of `86400` in the same way? I like the verbosity in line 29 but don't want it to be inconsistent with the others.

One thing we could do for brevity is define a variable like `const oneDayInMS = 60 * 60 * 24;` above the route declaration, then calculate off of that for the five values here that use it.

I'll try and swap this out today (when I finish the changelogs, too).